docs/Concepts/Data-Storage-Formats.md

```diff
@@ -0,0 +1,59 @@
+---
+description: Data storage formats
+---
+
+# Data storage formats
+
+Besu offers two formats for storing the world state, [Forest of Tries](#forest-of-tries) and [Bonsai Tries](#bonsai-tries).
+
+## Forest of Tries
+
+Forest of Tries, also called forest mode, is the [default storage format](../HowTo/Configure/Configure-Data-Storage.md).
+
+In forest mode, each node in the trie is saved in a key-value store by hash. For each block, the world state is updated
+with new nodes, leaf nodes, and a new state root. Old leaf nodes remain in the underlying data store. Data is accessed
+and stored by hash, which increases the size of the database and increases the resources and time needed to access account data.
+
```

Comment:
I think we should have a simplified diagram here to help readers visualize Forest of Tries. Same with the Bonsai Tries section. I can help brainstorm and show you how to use Figma next week.

Reply:
I have two diagram images from the slide show that might work.  I'll send them over.